src/common/config.go

```diff
@@ -74,6 +74,7 @@ type FeaturesConfig struct {
 	Reuse_cgroups       bool `json:"reuse_cgroups"`
 	Import_cache        bool `json:"import_cache"`
 	Downsize_paused_mem bool `json:"downsize_paused_mem"`
+	Enable_seccomp      bool `json:"enable_seccomp"`
```

Comment:
should we have a sub structure with options that get passed directly to the container?